web/instances/feature.go

```diff
@@ -71,6 +72,51 @@ func putFeatureSets(c echo.Context) error {
 	return c.JSON(http.StatusOK, inst.FeatureSets)
 }
 
+func getContextFromConfig(context string) (interface{}, error) {
+	contexts := config.GetConfig().Contexts
+
+	if context != "" {
+		ctx, ok := contexts[context]
+		if ok {
+			return ctx, nil
+		}
+	}
+
+	ctx, ok := contexts[config.DefaultInstanceContext]
+	if ok && ctx != nil {
+		return ctx, nil
+	}
+
+	return nil, fmt.Errorf("Unable to get context \"%s\" from config", context)
```

Comment:
[nit] You can `%q` instead of `"%s"` to print a quoted string

Reply:
ok, thanks.